### PR TITLE
Graylog integration test

### PR DESCRIPTION
**Pull request by YooSunYoung**
Docker services related to graylog is also added to the docker file.

### CODE REVIEW

File: tests/docker-compose-file-writer.yml [hunk 1]
```diff
@@ -67,5 +67,67 @@ services:
...
+    ulimits:
+      memlock:
+        hard: -1
+        soft: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+    restart: "on-failure"
+    networks:
+        - frontend
```
[the remark this answers is not in the document]
Comment (YooSunYoung) [reply tests/docker-compose-file-writer.yml:98]: Cuz Graylog is dependent on the `mongo db` and `opensearch`.
That's what I thought... can it be deployed without them...?

File: tests/docker-compose-file-writer.yml
[the remark this answers is not in the document]
Comment (YooSunYoung) [reply tests/docker-compose-file-writer.yml:1]: Exactly, I just wanted to test the github action if graylog can be started : D...
And I don't think we have to check if the log went through.
Testing the connection should be enough, no...?
And maybe checking if the configure works : D

[the remark this answers is not in the document]
Comment (jl-wynen) [reply tests/docker-compose-file-writer.yml:1]: Does this test the connection? How can you tell when a connection could not be made?

[the remark this answers is not in the document]
Comment (YooSunYoung) [reply tests/docker-compose-file-writer.yml:1]: No this PR is only about starting the graylog in the integration test environment.
Test is not added yet.

File: tests/docker-compose-file-writer.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (nitrosx) [reply tests/docker-compose-file-writer.yml:98]: I verified. Graylog needs both to run

**Review by nitrosx — APPROVED** [review]

**Pull request merged**